src/docs/app-monitoring/sysdig-monitor-create-alert-channels.md

```diff
@@ -21,85 +21,89 @@ sort_order: 3
+4. Attach the following script into Editor. **Note:** This is just an example; please update accordingly based on your use case:
 
-2. Click `Notification Channels` and `Add Notification Channel`. Choose `Custom Webhook` as the type
-   
-3. Use the webhook URL generated from Rocket.Chat and configure the notification channel. Name the channel in the format of `Rocketchat-alert-channel-<APP_TEAM_NAME>`
-
-4. Attach the following script into Editor, **NOTE** This is just an example, please update accordingly based on your use case:
 ```
-{
-        "text": "Sysdig Notification",
-        "attachments": [{
-          "title": " {{@event_title}}",
-          "title_link": "{{@alert_url}}",
-          "pretext": " {{@alert_description}}",
-          "color": "#f93b10",
-           "fields": [
-            {
-              "title": "State",
-              "short": true,
-              "value": "{{@event_state}}"
-            },
-            {
-              "title": "Alert level",
-              "short": true,
-              {{#if_severity_high}}
-             	 "value": "incident"
-               {{#else}}
-               	"value": "warning"
-               {{/if}}
-            },
-            {
-              "title": "Condition",
-              "short": false,
-              "value": "{{@alert_warning_condition}}"
-            },
-            {
-              "title": "Link to the Alert",
-              "short": false,
-              "value": "{{@alert_url}}"
-            }
-          ]
-      }]
-}
+   {
+     "text": "Sysdig Notification",
+     "attachments": [
+       {
+         "title": "{{@event_title}}",
+         "title_link": "{{@alert_url}}",
+         "pretext": "{{@alert_description}}",
+         "color": "#f93b10",
+         "fields": [
+           {
+             "title": "State",
+             "short": true,
+             "value": "{{@event_state}}"
+           },
+           {
+             "title": "Alert level",
+             "short": true,
+             {{#if_severity_high}}
+             "value": "incident"
+             {{#else}}
+             "value": "warning"
+             {{/if}}
+           },
+           {
+             "title": "Condition",
+             "short": false,
+             "value": "{{@alert_warning_condition}}"
+           },
+           {
+             "title": "Link to the Alert",
+             "short": false,
+             "value": "{{@alert_url}}"
+           }
+         ]
+       }
+     ]
+   }
 ```
 
-5. To test this notification, you have to pick one of the  **Select Alert Types** before you can **Send Test Notification**
+5. To test this notification, you have to pick one of the  **Select Alert Types** before you can **Send Test Notification**. Send test notification should send you a notification example in your channel if everything configured correctlly
 
 6. After tested, Click `Save` and now you should be able to see this notification option available in alert
 
```

Comment:
I still don't get a working webhook notification after working through these steps. I'm not sure if there is some mistake I'm making, or point I'm missing, but this section might need more work to be ready.

Reply:
My mistake here, forgot to 'enable' the integration which is now also added into the instructions.